webapp/test/test_updates.py

```diff
@@ -169,3 +169,72 @@ def test_process_list_empty_json(self):
             # NOTE: use copy of dict with json input, because process_list changes this dict
             self.updates_api.process_list(1, UPDATES_JSON_EMPTY.copy())
         assert "'package_list' is a required property" in str(context)
+
+class TestSplayTree(TestBase):
+    """ Set of tests for Splay tree implementation. """
+    hot_cache = HotCache()
+    test_nevra = []
+
+    @pytest.fixture(autouse=True)
+    def setup_dummy_data(self):
```

Comment:
Again, it's easier to directly set `test_nevra` (better `TEST_NEVRA`) collection in class head than append it later in method.